.github/workflows/pr.yml

```diff
@@ -24,22 +24,9 @@ jobs:
         diff_count=$(git status --porcelain=v1 2>/dev/null | wc -l)
         exit $diff_count
     - name: Build tools and unit tests
-      run: ant -f tlatools/org.lamport.tlatools/customBuild.xml compile compile-test dist
+      run: ant -f tlatools/org.lamport.tlatools/customBuild.xml info compile compile-test dist
     - name: Run unit tests
-      run: |
-        set +e # Ensure UT failure does not immediately terminate CI
-        set -o pipefail # Ensure UT failure is propagated through tee command
-        ant -f tlatools/org.lamport.tlatools/customBuild.xml test 2>&1 | tee test-output.txt
-        ut_exit_code=$?
-        echo "Unit test exit code: $ut_exit_code"
-        echo "UT_EXIT_CODE=$ut_exit_code" >> $GITHUB_ENV # Export UT result to CI env var
-    - name: Summarize unit tests
-      run: |
-        echo "Unit test exit code: $UT_EXIT_CODE"
-        python .github/scripts/parse-unit-test-output.py        \
-          test-output.txt                                       \
-          tlatools/org.lamport.tlatools/target/surefire-reports
-        exit $UT_EXIT_CODE
+      run: ant -f tlatools/org.lamport.tlatools/customBuild.xml -Dtest.halt=true test
```

Comment:
When a test fails, is there a way to get its output from the `target/surefire-reports/` directory? `parse-unit-test-output.py` may not be perfect, but (when it works) it does make the test output available in the CI log.

I was sad not to have that output for [one of the recent flaky test failures](https://github.com/tlaplus/tlaplus/actions/runs/9487536859/attempts/1) (#943), and I was considering making `parse-unit-test-output.py` part of the build in `main.yml` so we would have that output in the future.

Reply:
Perhaps, we should set `showoutput` to `yes`.

Reply:
https://ant.apache.org/manual/Tasks/junitreport.html

Reply:
@Calvin-L This is what a JunitReport looks like for a test suite run that was aborted after the first test failures:

<img width="1683" alt="image" src="https://github.com/tlaplus/tlaplus/assets/88777/6e28e4ce-a96a-49ea-9cf1-f229cd450daa">

Reply:
This seems like even nicer reporting for the pr.yml: https://github.com/marketplace/actions/junit-report-action#sample-%EF%B8%8F

Reply:
<img width="1666" alt="image" src="https://github.com/tlaplus/tlaplus/assets/88777/15c0ef86-780a-4f89-81c4-6e0b6024fa0b">

Reply:
The individual TEST-*.xml as well as the aggregated TESTS-*.xml can be imported in Eclipse.  I imagine similar support exists in IntelliJ and other IDEs:

<img width="446" alt="Screenshot 2024-06-17 at 9 35 35 PM" src="https://github.com/tlaplus/tlaplus/assets/88777/08445230-dd77-467a-bbba-8719703aa6c9">

Reply:
I believe this can be resolved.